### PR TITLE
[WIP] Translations

### PR DESCRIPTION
**Pull request by simonschaufi**
## Description

Add help messages to form fields

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] My code follows the code style
- [x] All files have a license header 
- [x] All methods have a doc header with type declarations
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Comment by simonschaufi** [comment]
Is this the way to go? Then I will add some more...

File: src/Form/TimesheetEditForm.php [hunk 1]
```diff
@@ -19,12 +19,27 @@
 use App\Entity\Timesheet;
 use App\Form\Type\ActivityGroupedWithCustomerNameType;
 use App\Repository\ActivityRepository;
+use Symfony\Component\Translation\TranslatorInterface;
```
Comment (kevinpapst) [inline src/Form/TimesheetEditForm.php]: can be removed

File: src/Form/TimesheetEditForm.php [hunk 2]
```diff
@@ -19,12 +19,27 @@
...
 
 /**
  * Defines the form used to manipulate Timesheet entries.
  */
 class TimesheetEditForm extends AbstractType
 {
+    /**
```
Comment (kevinpapst) [inline src/Form/TimesheetEditForm.php]: this complete new block can be removed

File: src/Form/TimesheetEditForm.php [hunk 3]
```diff
@@ -72,6 +87,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 },
             ])
             ->add('description', TextareaType::class, [
+                'help' => $this->translator->trans('timesheet.description.help'),
```
Comment (kevinpapst) [inline src/Form/TimesheetEditForm.php]: 'help' => 'timesheet.description.help',

Comment (kevinpapst) [reply src/Form/TimesheetEditForm.php]: Translation will automatically happen in the template if you added the translations to message.xx.xliff

**Comment by kevinpapst** [comment]
Nice idea!
And no, I always edit the files by hand ... which is a tedious task. A good editor would be nice.
Haven't looked if there is a free cloud solution for open source projects

File: src/Form/TimesheetEditForm.php (hunk 1 above)
Comment (simonschaufi) [reply src/Form/TimesheetEditForm.php]: Nop. It has to be included for the correct namespace.

File: src/Form/TimesheetEditForm.php (hunk 2 above)
Comment (simonschaufi) [reply src/Form/TimesheetEditForm.php]: why?

File: src/Form/TimesheetEditForm.php (hunk 3 above)
Comment (simonschaufi) [reply src/Form/TimesheetEditForm.php]: great. The documentation on symfony website is not very clear about that.

**Comment by kevinpapst** [comment]
I don't know what the fuck happened here. Somelse added a PR from the same branch and I just resolved the conflicts. Somehow Github seems to have merged that into master afterwards and I don't know why... damned, sorry!

**Comment by simonschaufi** [comment]
I don't understand. This pr is just from me

**Comment by kevinpapst** [comment]
Someone opened a PR and used your branch for it. But actually he just attached a ZIP (which I found out too late). I clicked "resolve conflicts" as I didn't see the attached file first and after I resolved the conflicts the branch was merged and both PR were closed.
I have no clue why that happened, feels like a GitHub bug. The other PR was about translations as well, thats why I didn't see that it was using your branch. I guess this PR was closed, because the other PR had your branch attached.

Nevertheless the change was exactly one line, so not much is lost ;-) But now there is an untranslated  string in the UI, if you want to start off again from there:
https://github.com/kevinpapst/kimai2/commit/8b8583d48b60693ad090b55f300c739f3aa13112

**Pull request merged**